.github/workflows/e2e-tests-main-devnet.yml

```diff
@@ -32,78 +33,22 @@ jobs:
       - name: Checkout Source code
         uses: actions/checkout@v2
 
-      - name: Install sccache for ubuntu-latest
-        env:
-          LINK: https://github.com/mozilla/sccache/releases/download
-        run: |
-          SCCACHE_FILE=sccache-$SCCACHE_VERSION-x86_64-unknown-linux-musl
-          mkdir -p $HOME/.local/bin
-          curl -L "$LINK/$SCCACHE_VERSION/$SCCACHE_FILE.tar.gz" | tar xz
-          mv -f $SCCACHE_FILE/sccache $HOME/.local/bin/sccache
-          echo "$HOME/.local/bin" >> $GITHUB_PATH
-
-      - name: Restore Cargo Cache
-        uses: actions/cache@v2
-        with:
-          path: |
-            ~/.cargo/bin/
-            ~/.cargo/registry/index/
-            ~/.cargo/registry/cache/
-            ~/.cargo/git/db/
-            target/
-          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
-
-      - name: Install Rust Toolchain
-        uses: actions-rs/toolchain@v1
-        with:
-          profile: minimal
-          toolchain: ${{ env.RUST_TOOLCHAIN_VERSION }}
-          override: true
-
-      - name: Install WASM target
-        run: rustup target add wasm32-unknown-unknown --toolchain "$RUST_TOOLCHAIN_VERSION"
-
-      - name: Restore sccache
-        uses: actions/cache@v2
-        continue-on-error: false
-        with:
-          path: ${{ env.SCCACHE_PATH }}
-          key: ${{ runner.os }}-sccache-${{ hashFiles('**/Cargo.lock') }}
-
-      - name: Start sccache server
-        run: sccache --start-server
-
-      - name: Build Binary
-        run: |
-          export RUSTC_WRAPPER=$HOME/.local/bin/sccache
-          cargo build --release
-
-      - name: Build runtime
-        run: |
-          export RUSTC_WRAPPER=$HOME/.local/bin/sccache
-          cargo build --release -p aleph-runtime
-
-      - name: Print sccache stats
-        run: sccache --show-stats
+#      - name: Restore Cargo Cache
+#        uses: actions/cache@v2
+#        with:
+#          path: |
+#            ~/.cargo/bin/
+#            ~/.cargo/registry/index/
+#            ~/.cargo/registry/cache/
+#            ~/.cargo/git/db/
+#            target/
+#          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
```

Comment:
Enabling caching reduces docker image size - the issue has been already raised on Slack